mmnotes.txt

```diff
@@ -4890,7 +4890,7 @@ http://planetx.cc.vt.edu/AsteroidMeta/metamath ).  It has actually been
 modernized slightly, to remove the requirement that the empty set exist.
 This eliminates the need for the Axiom of Replacement, from which
 empty set existence is derived.  The original can be seen at
-http://de2.metamath.org/metamath/set.mm .
+https://de.metamath.org/metamath/set.mm .
```

Comment:
It seems like the new link doesn't work

Reply:
Both old and new versions are broken... It's probably lost forever, and not a big loss. I'd simply remove that sentence, but this is in old notes anyway. I'd leave it like that in this MR.

Reply:
Sounds like it is in the git history somewhere, if I'm correctly following. But it doesn't seem especially important to have a specific link in `mmnotes.txt` so just removing the sentence seems fine (at least for now, probably indefinitely).